doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -166,7 +166,11 @@ The standard Fast Pair API (without extensions) currently supports the :c:func:`
 
 The standard Fast Pair (without extensions) does not require registration of any callback type, meaning all callbacks are optional.
 
-Apart from the callback registration, no additional operations are needed to integrate the standard Fast Pair implementation.
+After the callback registration, the Fast Pair subsystem must be enabled with the :c:func:`bt_fast_pair_enable` function.
+The Fast Pair subsystem readiness can be checked with the :c:func:`bt_fast_pair_is_ready` function.
+The Fast Pair subsystem can be disabled with the :c:func:`bt_fast_pair_disable` function.
```

Comment:
Consider also mentioning the effects of having the subsystem in the disabled state:

- One sentence here: `In the Fast Pair subsystem disabled state, most Fast Pair APIs are not available`

- One sentence in the Interacting with GATT service section. For example: `In the Fast Pair subsystem disabled state, GATT operations on the Fast Pair service are rejected`.